catkit/hardware/thorlabs/ThorlabsTSP01.py

```diff
@@ -104,27 +106,26 @@ def get_error_message(cls, status_code):
     def create(cls, config_id):
         return cls(config_id=config_id, serial_number=CONFIG_INI.get(config_id, "serial_number"))
 
-    @classmethod
-    def find_all(cls):
+    def find_all(self):
```

Comment:
This was needed such that ``self.instrument_lib.TLTSPB_findRsrc`` had a instance populated ``serial_number``.